draft-iab-path-signals-collaboration.md

```diff
@@ -542,6 +542,12 @@ topics would be welcome.
   stored or what purpose information may be used for.  Similar issues
   may also be applicable to the kind of information sharing discussed
   in this document.
+  
+* Policy and technology need to fit each other. The present work has
```

Comment:
I would rather remove the first sentence here. That's very general...

Reply:
Ok

Reply:
Done